06.bowling_object/game.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require './frame'
+
+class Game
+  def initialize
+    @marks = ARGV[0].split(',')
```

Comment:
```suggestion
    @shots = ARGV[0].split(',').map { |mark| Shot.new(mark) }
```

として最初からオブジェクトにした方がコードがスッキリしそうです

https://bootcamp.fjord.jp/pages/utilize-objects